File: Python/stack.py

class Stack:
    def __init__(self ):
        self.__data__ = [] 

    def push_item (self, item):
        self.__data__.append(item)

    def pop_item (self):
        return self.__data__.pop()

    def get_stack(self):
        return self.__data__ 


if __name__ == "__main__":

     S = Stack()

     S.push_item("A")
     S.push_item("B")
     S.push_item("C")
     S.push_item("D")
         
     print(S.get_stack())

     print(S.pop_item())       
  
     print(S.get_stack())

File: CPP/stack.cpp
#include<stdio.h>
#include <stack>
using namespace std;

int main (int argc, char *argv[]){
   stack<int> s;


   s.push(1);
   s.push(2);
   s.push(2);
   s.push(3);
   s.push(4);

   s.pop();


   s.top();

}

